### DISCUSSION TITLE
Add NetStandard2.0 as a target for a new package xunit.v3.core.abstractions

### DISCUSSION THREAD

**Discussion by SteveBush** [xunit/xunit#3090] [Ideas]
I have a set of cross-platform test utility libraries that extend the functionality of xUnit v2.  For the shared libraries, I target netstandard2.0, net6.0 and net9.0.  I have test classes that are platform-specific (net9.0-windows;net6.0-windows;netcoreapp3.1;net472;), which target legacy and current implementations of .net.

In v2, I used the package xunit.abstractions targeting netstandard2.0 in my test libraries and the appropriate xunit test runner package in my test classes.  

For v3, it would be awesome to have a package targeting netstandard2.0 for library development, that includes the contents of xunit.v3.core.abstractions.

**Comment by bradwilson** [comment]
I am confused. There is no such library/package named `xunit.v3.core.abstractions`. Are you suggesting that we extract some things from `xunit.v3.extensibility.core` into a new library? What would be the benefit over just taking the `xunit.v3.extensibility.core` dependency?

**Comment by SteveBush** [comment]
My bad.  I missed xunit.v3.common and xunit.v3.extensibility.core both target netstandard2.0 and have the interface abstractions I need.  Sorry the waste of time.
